src/content/lodash/function/bind/vanilla.js

```diff
@@ -0,0 +1,17 @@
+const bind = (fn, ctx, ...boundArgs) => {
+  return (...args) => {
+    const jointArgs = [...boundArgs, ...args]
+    return fn.call(ctx, ...jointArgs)
+  }
```

Comment:
Just curious, wouldn't it be easier just to use native `bind`? Is there any caveats?

```suggestion
  return fn.bind(ctx, ...boundArgs)
```

Reply:
I have to be completely honest, I just didn't think of it 😅